medicines/web/components/search-results/pagination.tsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import styled from 'styled-components';
+import { black, mhraBlue } from '../../styles/colors';
+import { mobileBreakpoint } from '../../styles/dimensions';
+
+const StyledPagination = styled.nav`
+  ul {
+    display: flex;
+    justify-content: space-between;
+    list-style: none;
+    margin: 40px auto 20px;
+    max-width: 80%;
+    padding: 0;
+  }
+
+  a {
+    color: ${mhraBlue};
+  }
+
+  .arrow a {
+    color: ${black};
+    text-decoration: none;
+  }
+
+  .pagination-number,
+  .middle-group {
+    display: flex;
+    flex-wrap: nowrap;
+  }
+
+  .pagination-number li,
+  .middle-group li {
+    margin-right: 0.5rem;
+  }
+
+  li span {
+    padding-left: 0.5rem;
+  }
+
+  @media ${mobileBreakpoint} {
+    font-size: 0.875rem;
+  }
+`;
+
+const getPaginationGroups = (pageCount: number, currentPage: number) => {
```

Comment:
Probably worth defining an interface for the return type. This makes it easier down the line to prevent unintended changes and also easier for intellisense in various editors.

Reply:
interface `IPaginationGroups` created